hpx/runtime/naming_fwd.hpp

```diff
@@ -29,12 +29,6 @@ namespace hpx
 
         HPX_API_EXPORT resolver_client& get_agas_client();
 
-        using component_type = std::int32_t;
-        using address_type = std::uint64_t;
-
-        HPX_CONSTEXPR_OR_CONST std::uint32_t invalid_locality_id =
-            ~static_cast<std::uint32_t>(0);
-
```

Comment:
Could we include `hpx/naming_base.hpp` here?

Reply:
yep !